Lib/test/test_pathlib.py

```diff
@@ -1778,6 +1778,15 @@ def test_link_to(self):
         self.assertEqual(os.stat(r).st_size, size)
         self.assertTrue(q.stat)
 
+    @unittest.skipIf(hasattr(os, "link"), "os.link() is present")
+    def test_link_to_failure(self):
```

Comment:
Would not `test_link_to_not_implemented` be better name?

Reply:
Fixed, and added :)